### PR TITLE
8691 summernote list extra space bug

### PR DESCRIPTION
**Pull request by arisah-ijeoma**

### CODE REVIEW

File: app/assets/stylesheets/admin/loans/_questionnaires.scss [hunk 1]
```diff
@@ -71,6 +71,10 @@ section.questionnaires {
     margin-bottom: 1em;
   }
 
+  .note-editable li, .answer li {
+    min-height: 1em;
```
Comment (meltheadorable) [inline app/assets/stylesheets/admin/loans/_questionnaires.scss]: you may also want to set the margin-bottom to `0` here, i don't think list items should have quite that much space between them?

File: app/views/admin/loans/questionnaires/_questionnaire_group.html.slim [hunk 2]
```diff
@@ -84,7 +84,8 @@
               = f.input_field :"#{question.attribute_sym}[number]", as: :decimal,
                 value: response.number
 
-            - tags = %w(table tbody tr td b i u ol ul p)
+            - tags = %w(table tbody tr td b i u ol ul li p)
+            = response.text
```
Comment (meltheadorable) [inline app/views/admin/loans/questionnaires/_questionnaire_group.html.slim]: i believe this is a debugging line, you'll want to remove it

File: app/views/admin/loans/questionnaires/_questionnaire_group.html.slim [hunk 3]
```diff
@@ -84,7 +84,8 @@
               = f.input_field :"#{question.attribute_sym}[number]", as: :decimal,
                 value: response.number
 
-            - tags = %w(table tbody tr td b i u ol ul p)
+            - tags = %w(table tbody tr td b i u ol ul li p)
```
Comment (meltheadorable) [inline app/views/admin/loans/questionnaires/_questionnaire_group.html.slim]: Great catch!

File: app/views/admin/loans/questionnaires/_questionnaire_group.html.slim (hunk 2 above)
Comment (arisah-ijeoma) [reply app/views/admin/loans/questionnaires/_questionnaire_group.html.slim]: thanks for the find

**Pull request merged**